src/plugins/Network.js

```diff
@@ -37,6 +37,14 @@ module.exports = class Network extends OverlayPlugin {
         this.renderToolbarButton();
         this.renderWidget();
 
+        /*
+         * Redis message for enabling/disabling mobile throttling and 5G support
+         * could be sent without rendering the widget from srach
+         * to avoid recreation of widget elements, check this parameters before.
+         */
+        this.mobileThrottlingConfigured = false;
```

Comment:
using mobileThrottlingEnabled and network5GEnabled, you would be able to have a more future proof behavior by being able to toggle them dynamically. Here you are stuck with a configuration, and not able to modify widget dynamically. (when configuration would have change on the device for example)